sunpy/data/sunpyrc

```diff
@@ -35,12 +35,11 @@ time_format = %Y-%m-%d %H:%M:%S
 ; Location to save download data to. Path should be specified relative to the
 ; SunPy working directory.
 ; Default value: data/
-;download_dir = /tmp
 download_dir = data
 
-; Location where the sample data will be downloaded. Path should be specified
-; relative to the SunPy working directory.
-sample_dir = data/sample_data
+; Location where the sample data will be downloaded. If not specified, will be
+; downloaded to platform specific user data directory.
```

Comment:
could you add some examples here for different systems?

Reply:
Should we put that in the sunpyrc  file and not maybe a doc string?

Reply:
You mean like where they will be put by default? I was assuming that this file isn't a way to document the defaults, and that if the user is here changing this they are not going to care to set it to something close to the default?

Reply:
From a user's perspective I would be annoyed because if I want to know this I now have to search. Maybe just link to the package that does this?

Reply:
I am ok linking it here. I think I did at the top of the file when I switched to appdirs.

Reply:
I'm fine with this PR once and also likely an update to the docs page on sample data as well.

Reply:
The docs currently state:

> This will download the data to your sample-data directory which can be customized by editing the sunpyrc file (see Customizing SunPy).

Which still holds true, so I think they are ok.